sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BasicWriteStatsTracker.scala

```diff
@@ -173,7 +173,7 @@ object BasicWriteJobStatsTracker {
     val sparkContext = SparkContext.getActive.get
     Map(
       NUM_FILES_KEY -> SQLMetrics.createMetric(sparkContext, "number of written files"),
-      NUM_OUTPUT_BYTES_KEY -> SQLMetrics.createMetric(sparkContext, "bytes of written output"),
+      NUM_OUTPUT_BYTES_KEY -> SQLMetrics.createSizeMetric(sparkContext, "written output"),
```

Comment:
This change looks reasonable to me, but it seems there are other places having the same issue?, e.g.,
https://github.com/apache/spark/blob/a77505d4d3e5db4413d7fa76610265792d949c64/sql/core/src/main/scala/org/apache/spark/sql/execution/basicPhysicalOperators.scala#L666

Reply:
Thanks. I went through the code and corrected it in a few more places.